dashboard/test/controllers/api/v1/users_controller_test.rb

```diff
@@ -119,4 +119,29 @@ class Api::V1::UsersControllerTest < ActionController::TestCase
     test_user.reload
     assert_equal response["next_census_display"], test_user.next_census_display
   end
+
+  test "a get request to get school_name returns school object" do
+    sign_in(@user)
+    get :get_school_name, params: {user_id: 'me'}
```

Comment:
it's not clear to me how this is passing. Why is the string `'me'` a valid user id?

Reply:
In the controller "me" is the current signed in user.  I will update the test to include an integer, which is consistent with the data type required.